druid-derive-data/Cargo.lock

```diff
@@ -0,0 +1,47 @@
+# This file is automatically @generated by Cargo.
```

Comment:
I don't think we need `Cargo.lock` to be checked in; this is usually only done for binary crates.

Reply:
Oups, I did not know!